runmariadb

```diff
@@ -32,7 +33,7 @@ if [[ -f "$MARIADB_CERT_FILE" && "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "
 fi
 
 if [ ! -d "${DATADIR}/mysql" ]; then
-    crudini --set "$MARIADB_CONF_FILE" mysqld max_connections 64
+    crudini --set "$MARIADB_CONF_FILE" mysqld max_connections "$MARIADB_MAX_CONNECTIONS"
```

Comment:
Please add curlys braces to follow convention.

Reply:
thx, add curlys braces